src/app/docs/definition-list/definition-list-docs.component.html

```diff
@@ -0,0 +1,19 @@
+<sky-docs-demo-page
+  moduleName="SkyDefinitionListModule"
+  moduleSourceCodePath="src/app/public/modules/definition-list/"
+  pageTitle="Definition List"
+>
+  <sky-docs-demo-page-summary>
+    The definition list module displays a list of label-value pairs.
+  </sky-docs-demo-page-summary>
+
+  <sky-docs-design-guidelines>
+    <sky-docs-demo-page-section
+      heading="UX guidelines"
+    >
+      <p>
+        Don't append colons to definition list labels.
```

Comment:
Re-reviewing this, it occurs to me that we are unnecessarily using a 2-dollar word when more common terminology would make things clearer. I'd rewrite to avoid `append`.
```suggestion
        Don't use colons after definition list labels.
```